src/components/SummaryBox/SummaryBoxItem.tsx

```diff
@@ -6,8 +6,8 @@ import CardBody from '../Card/CardBody';
 
 interface SummaryBoxItemProps extends Omit<CardProps, 'color' | 'outline' | 'className'> {
   className?: string;
-  label?: string;
-  value?: React.ReactNode;
+  label?: string | React.ReactNode;
+  value?: string | React.ReactNode;
```

Comment:
Isn't `ReactNode` already a type union with `string`? Maybe different versions of `@types/react` are different.

Reply:
You are right, let me remove `string`